target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -199,6 +199,9 @@ ubnt,nanostation-ac)
 	ucidef_set_led_rssi "rssimediumhigh" "RSSIMEDIUMHIGH" "ubnt:blue:rssi2" "wlan0" "51" "100"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "ubnt:blue:rssi3" "wlan0" "76" "100"
 	;;
+ubnt,acb-isp)
+	ucidef_set_led_default "system" "SYSTEM" "white:status" "1"
```

Comment:
Sorry, but while going through the changes here locally again, I've noticed, that I've missed following previously.

1. The name of the LED should follow boardname(or vendor):color:function, so it should be `ubnt:white:status`
2. This line is superfluous, as you're doing already the same in DTS

Reply:
1. OK. I can change that.
2. Does that mean that I should just completely remove the "ucidef_set_led_default" line?

Reply:
2. Yes. Note that there is not a single ucidef_set_led_default in 01_leds for ath79.

Reply:
Hello @adrianschmutzler,
thanks for the answer. I already discussed that with ynezz a bit farther down. Currently I still have the problem that the SPI is too fast on the device. I started a discussion (and didn't receive a reply for two weeks until I pinged the subsystem maintainer) on the Linux SPI mailing list. The Maintainer suggested a rough direction how I could slow down the driver without having too much performance impact on other users. I wanted to take a look at that (most likely) this weekend. If I have something that is acceptable for Linux, I'll update the patch set here - including the changes already suggested by ynezz.